### DISCUSSION TITLE
CONFIGURATION MODE  ERROR

### DISCUSSION THREAD

**Discussion by RaySly** [Mustard2/MustardUI#110] [Help]
Hello! :) here with a new problem, when I enter in configuration mode and want to add custom properties the MustardUI menu does not appear for any option, that is, if I want to add a shape key or a value of a material or a mask or whatever, the menu that before if it came out now no longer x_x
![imagen_2023-08-25_153730574](https://github.com/Mustard2/MustardUI/assets/44752357/7105ca52-ca1c-47da-84ca-ac65a9938d08)

**Comment by Mustard2** [comment]
Sorry I also missed this. Might be a bug, but are you sure you are in Configuration mode? I don't see MustardUI panel, can you try clicking on the armature?

**Comment by Mustard2** [comment]
Closing this as it's very old.

Please open a new Issue if the this is still happening
